.github/workflows/documentation.yaml

```diff
@@ -15,28 +16,35 @@ jobs:
       fail-fast: false
       matrix:
         module:
```

Comment:
can we automate the discovery of the folders some how
as this will go stale really quickly